.circleci/config.yml

```diff
@@ -0,0 +1,143 @@
+version: 2.1
+
+aliases:
+  - &working_directory ~/merchant-center-application-kit
+
+  - &restore_yarn_cache
+    name: 'Restoring yarn cache'
+    keys:
+      - v1-yarn-cache-{{ checksum "yarn.lock" }}
+      - v1-yarn-cache
+
+  - &save_yarn_cache
+    name: 'Saving yarn cache'
+    key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
+
+executors:
+  node_11:
+    docker:
+      - image: circleci/node:11.10
+    working_directory: *working_directory
+
+orbs:
+  cypress: cypress-io/cypress@1
+
+commands:
+  install_dependencies:
+    description: 'Installing dependencies'
+    steps:
+      - checkout
+      - restore_cache: *restore_yarn_cache
+      - run: yarn install --pure-lockfile
+      - save_cache: *save_yarn_cache
+  build_artefacts:
+    description: 'Building artefacts'
+    steps:
+      - run: yarn build
+  lint:
+    description: 'Running linters'
+    steps:
+      - run: yarn run jest --projects jest.eslint.config.js jest.stylelint.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  unit_test:
+    description: 'Running tests'
+    steps:
+      - run: yarn run jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  vrt_test:
+    description: 'Running Visual Regression Tests'
+    steps:
+      - run: yarn run-if:percy
+  build_playgrond:
```

Comment:
`build_playground`

Reply:
Will go anyway due to CircleCI orbs. I am still working on it.